infra/templates/backend-state-setup/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+name="cobaltfstate"
```

Comment:
As per our discussion, are we planning on preferring `variables.tf` or CLI args to `*.tfvars`?